buildpacks/maven/src/main.rs

```diff
@@ -201,30 +209,51 @@ impl Buildpack for MavenBuildpack {
         // running since they might be confusing to the user.
         let internal_maven_options = vec![String::from("-B")];
 
-        log_header("Executing Maven");
-        log_info(format!(
-            "$ {} {} {}",
-            mvn_executable.to_string_lossy(),
-            shell_words::join(&maven_options),
-            shell_words::join(&maven_goals)
-        ));
-
-        util::run_command(
-            Command::new(&mvn_executable)
+        output::print_section("Running Maven build");
+        output::print_subsection(BuildpackOutputText::new(vec![
+            BuildpackOutputTextSection::regular("Running "),
+            BuildpackOutputTextSection::value(format!(
+                "{} {} {}",
+                mvn_executable.to_string_lossy(),
+                shell_words::join(&maven_options),
+                shell_words::join(&maven_goals)
+            )),
+        ]));
```

Comment:
- Request: Log the command based off of what's going to be run. In this case the logging and the command can differ, and actually do here. You're running with `internal_maven_options` but are not printing them. Also you're using `maven_options.chain()` below but not here.

If that's intentional please comment it. Here's an example where I'm intentionally hiding command args from the user (though the full/real command will show on error/failure):

https://github.com/heroku/buildpacks-ruby/blob/187770e700a433cf9e6af678cb3eb5735b89eec5/buildpacks/ruby/src/layers/bundle_download_layer.rs#L99

Reply:
I moved this feedback to a "lster" issue https://github.com/heroku/buildpacks-jvm/issues/762